src/lib/utils/track/schema.ts

```diff
@@ -328,18 +333,20 @@ export enum ContextModules {
   ArtworkImage = "ArtworkImage",
   ArtworkTombstone = "ArtworkTombstone",
   Bibliography = "Bibliography",
+  Collection = "Collection",
+  CollectionDescription = "CollectionDescription",
   CommercialButtons = "CommercialButtons",
   ExhibitionHistory = "ExhibitionHistory",
+  FeaturedArtists = "FeaturedArtists",
   PartnerContext = "PartnerContext",
   Provenance = "Provenance",
-  Collection = "Collection",
-  CollectionDescription = "CollectionDescription",
```

Comment:
Seems like this list was previously alphabetized, restoring the status quo here.